webapp/src/main/webapp/js/individual/individualQtipBubble.js

```diff
@@ -6,48 +6,44 @@ $(document).ready(function(){
     
     $.extend(this, i18nStrings);
 
+    $('head').append('<style id="researchAreaCSS">.qtip { font-size: 14px; max-width: none !important; } .researchAreaTip { background: url(' + imagesPath + '/individual/researchAreaBubble.png) no-repeat; } </style>');
+    padding: '0em',
```

Comment:
I'm not sure it "should". But in the short term, it needed to be in order to replicate the embedded stylesheet code that was used with qTip, but isn't supported in qTip 2